tests/test_completion_refresher.py

```diff
@@ -1,87 +1,82 @@
 import time
-import pytest
+import unittest
 from mock import Mock, patch
-
-
-@pytest.fixture
-def refresher():
-    from mssqlcli.completion_refresher import CompletionRefresher
-    return CompletionRefresher()
-
-
-def test_ctor(refresher):
-    """
-    Refresher object should contain a few handlers
-    :param refresher:
-    :return:
-    """
-    assert len(refresher.refreshers) > 0
-    actual_handlers = list(refresher.refreshers.keys())
-    expected_handlers = ['schemata', 'tables', 'views',
-                         'types', 'databases', 'casing', 'functions']
-    assert expected_handlers == actual_handlers
-
-
-def test_refresh_called_once(refresher):
-    """
-
-    :param refresher:
-    :return:
-    """
-    callbacks = Mock()
-    pgexecute = Mock()
-
-    with patch.object(refresher, '_bg_refresh') as bg_refresh:
-        actual = refresher.refresh(pgexecute, callbacks)
+from mssqlcli.completion_refresher import CompletionRefresher
+
+class CompletionRefresherTests(unittest.TestCase):
+
+    def test_ctor(self):
+        """
+        Refresher object should contain a few handlers
+        :param refresher:
+        :return:
+        """
+        refresher = CompletionRefresher()
+        assert len(refresher.refreshers) > 0
+        actual_handlers = set(refresher.refreshers.keys())
+        expected_handlers = set(['databases', 'schemata', 'tables', 'types', 'views'])
```

Comment:
Is 'schemata' a typo?

Reply:
No. `schemata` is one of the refresher defined at `completion_refresher.py`

Reply:
I still think it's typo. Can we rename this to 'schemas' ?

Reply:
Fixed.